src/Castle.Core.Tests/MultipleRepeatedInterfaceTestCase.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+using Castle.DynamicProxy;
+using NUnit.Framework;
+
+namespace CastleTests
+{
+    [TestFixture]
+    public class MultipleRepeatedInterface

```

Comment:
Also, once you do, you'll see that the solution doesn't really fix the problem as the assembly it generates is invalid.